.github/ISSUE_TEMPLATE/add_dataset.yml

```diff
@@ -0,0 +1,79 @@
+name: Add dataset
+description: Suggest a new Library, Archive or Museum dataset to include in the Hugging Face Hub.
+title: "[Add Dataset]: "
+labels: ["dataset"]
+assignees: "davanstrien"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for suggesting a dataset!
+  - type: textarea
+    id: data-description
+    attributes:
+      label: Dataset description
+      description: A brief description of the dataset and why it might be of interest
+      placeholder: Please summarise the dataset and why it might be interesting to add to the hub. 
+      value: "This dataset is awesome! Here is why..."
+    validations:
+      required: true
+  - type: dropdown
+    id:  dataset-modality
+    attributes:
+      label: Dataset modality
+      description: What is the modality of the data?
+      options:
+        - Text
+        - Image
+        - Audio
+        - Video
+        - Mixed
+        - Other
+    validations:
+      required: true
+  - type: checkboxes
+    id: type-of-dataset
+    attributes:
+      label: Type of dataset
+      description: What type of datset is this?
+      options:
+        - label: A large unlabeled text corpus, i.e. something that could be used for training language models
+        - label: A large unlabeled image corpus, i.e. something that could be used for pre-training image models
+        - label: An annotated/labelled dataset, i.e. somethng that could be used to train/evaluate a machine learning model
+        - label: A benchmark dataset, i.e. a dataset that has been used to evaluate the performance of machine learning models
+        - label: Other
+    validations:
+      required: true
```

Comment:
I'm not sure about the pertinence of this field.

In Datasets we have the concept of "tasks" and a dataset can be used for multiple tasks. The list is long: see https://github.com/huggingface/datasets/blob/master/src/datasets/utils/resources/tasks.json

Reply:
My main idea here was to try and ensure people don't add things that are out of scope for machine learning (e.g. very small collections of text without labels). However, since we will probably do an initial check to validate proposed datasets we can catch any issues with datasets there. 

I suggest we remove this from this section and people can align with the "tasks" when creating loading scripts etc. WDYT?